src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -53,7 +54,7 @@
       @Override
       public TenantManager get() {
         tenantManagerCreated.set(true);
```

Comment:
This variable is no longer needed now that doDestroy is a no-op.

Reply:
Done